firefox_puppeteer/tests/manifest.ini

```diff
@@ -6,7 +6,8 @@
 [test_software_update.py]
 
 # UI tests
-[test_about_window.py]
+#TestAboutWindow.test_basic timing out 150817
+#[test_about_window.py]
```

Comment:
This is not how we disable tests. Please do it the proper way and also file a bug for a reference. We have to take a look at this.